traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -284,7 +284,7 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := Snapshot(inf.Tx.Tx, crConfig, tm); err != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), nil)
 		return
 	}
```

Comment:
Same here: An error returned from `crconfig.Snapshot()` *is* 500-level, not 400-level. The endpoint should check at an earlier point whether a CDN with the given name exists and return a 400 bad request at *that* point if it doesn't.